mandalab_visual/mandalab_visual.py

```diff
@@ -0,0 +1,77 @@
+import datetime
+from pyvis.network import Network
+from mandalab_visual.consts import *
+
+DEFAULT_GRAPH_NAME_FORMAT = "{graph_base}_{name}_{timestamp}.html"
+
+
+def guess_if_running_in_notebook():
+    try:
+        from IPython import get_ipython
+        if get_ipython() is None or 'IPKernelApp' not in get_ipython().config:
```

Comment:
maybe move this if outside the `try` block , so that you can check first if you can import and then your condition